go.mod

```diff
@@ -3,26 +3,28 @@ module sigs.k8s.io/kubefed
 go 1.14
 
 require (
-	github.com/evanphx/json-patch v4.5.0+incompatible
+	github.com/evanphx/json-patch v4.9.0+incompatible
 	github.com/ghodss/yaml v1.0.0
+	github.com/go-logr/logr v0.2.1 // indirect
+	github.com/go-logr/zapr v0.2.0 // indirect
 	github.com/json-iterator/go v1.1.10
 	github.com/onsi/ginkgo v1.14.0
 	github.com/onsi/gomega v1.10.1
 	github.com/pborman/uuid v1.2.0
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.7.1
-	github.com/spf13/cobra v0.0.5
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.6.1
-	k8s.io/api v0.18.6
-	k8s.io/apiextensions-apiserver v0.18.6
-	k8s.io/apimachinery v0.18.6
-	k8s.io/apiserver v0.18.6
-	k8s.io/client-go v0.18.6
-	k8s.io/component-base v0.18.6
+	k8s.io/api v0.19.2
```

Comment:
Now that `v1.19.3` is out we could update to `v0.19.3` here and other dependencies. Would you mind taking care of that?

Reply:
1.19.3 is not available in all components yet (eg. in k8s.io/api). I guess we can hold off merging until it's available.

Reply:
Ah yeah good point - thanks for checking that out.